content/soarian-financials/stu3/individuals/patient.md

```diff
@@ -0,0 +1,638 @@
+	 "text": "Married",
+	 "coding": [
+	 {
+	  "system": "http://hl7.org/fhir/v3/MaritalStatus",   
+	  "code": "M",
+	  "display": "Married"
+	  }
+			   ]
+					}
+
+#### Communication	
+
+  Preferred language which may be used to communicate with the patient about his or her health.
+
+  
+_Example_
+
+	"communication": [
+	{
+	 "language": {
+	 "coding": [
+	  {
+	  "system": "urn:ietf:bcp:47",
+	  "code": "en",
+	   "display": "ENGLISH"
+	  }
+			  ],
+	"text": "ENGLISH"
+				},
+	"preferred": true
+	}
+					]
+  
+	Preferred should always be true.
+  
+
+#### Address	
+
+  Addresses for the individual.
+
+  
+_Example_
+
+	"address": [
+	{
+	 "use": "HOME",
+	 "type": "POSTAL",
+	 "text": "51 valley stream parkway,Malvern,PA 19355",
+	 "line": [
+			  "51 valley stream parkway"
+			 ],
+	"city": "Malvern",
+	"district": "Chester",
+	"state": "PA",
+	"postalCode": "19355",
+	"country": "US",
+	"period": {
+			  "start": "2018-12-12"
+			  }
+	}
+			  ]
+  
+ 
+Only the following type and use are supported
+  
+ Type	 | Use     | Soarian Finanicals field
+-------- |---------|-------------------------------
+Postal   | Home    | Mailing address
+Physical | Home    | Residence address
+Physical | Temp    | Temporary Residence
+
+#### Deceased indicator	
+
+  Indicated if the patient is deceased
+  
+_Example_
+
+	{
+	 "deceasedBoolean": true,
+	}
+
+  
+#### Deceased date	
+
+  Indicated the date when the patient was deceased
+  
+_Example_
+
+	{
+	 "deceasedDateTime": "2018-03-01",
+	}
+
+  Only deceased date can be entered. Deceased time will not be saved.
+  
+  
+  
+#### Patient User data	
+
+  User data fields are client-specific fields unique to an organization that are not represented by Soarian Financials model fields. User data fields are created and maintained using the Additional Data master file available from Master File Central
+  
+
+_URL for this extension_
+
+	"extension": {
+	"url": "[ CORRECT FHIR SERVER]/StructureDefinition/patient-userData",
+		"extension": [{
+		 url": "[ CLIENT FHIR SERVER]/StructureDefinition/patient-userData#[user _data_field_name]", //R! URL that indicates the user data field name and description
+		 "valueString": "string" // R! Value of the user data field
+				 }]
+				 }
+
+  
+_Example_
+
+	extension": {
+	"url": "[Soarian Financials URL]/stu3/StructureDefinition/patient-userData",
+		"extension": [{
+		url": "http:// [ CLIENT FHIR SERVER]/StructureDefinition/MedhistoryConsent",
+		"valueString": "Y" 
+				  }]
+				}
+
+## Update
+
+Updates an individual Patient:
+
+    PUT /Patient/<Soarian Finanicals Person Number>
+
+_Implementation Notes_
+
+* The update folows the same rules and format as a create.
+* All the existing information for the patient needs to be sent irrespective of that information being updated or not.
+  
+
+[Time of day of birth]: http://hl7.org/fhir/DSTU2/extension-patient-birthtime.html
+[Patient.deceased]: http://hl7.org/fhir/patient-definitions.html#Patient.deceased_x_
```

Comment:
without specifying a version in the hl7.org URL, this will continuously link to newer and newer versions as they are released (hl7.org/fhir/x links to 'current')